docs/version_control/pull-request-review-process.md

```diff
@@ -0,0 +1,53 @@
+# PR Review guidelines
+
+## Step 1: Create a Pull Request
+
+Create a pull request with your changes. It's a good idea to create a `draft pull request` first, so that you can get feedback on your changes before you're ready ask for a review. It's also recommended to add the card number as a link to the pull request description, so that it's easy to find the card in the project management tool.
+
+## Step 2: Post in the PR Reviews Teams Channel
+
+Make a post in the `PR Reviews` Teams channel to let others know that you're ready for a review and move your card to `Pending Dev Review` column. It's highly recommended to include the following information:
+
+- The card number and link to the card in the project management tool
+- The link to the pull request
+- Tagging the `PR Reviews` team
+- Any additional context or information that would be helpful for reviewers
```

Comment:
Here we could have an example of "what is a good PR Reviews post"

Reply:
@pbastia Do you mean something like a screenshot from one of our existing PR review posts?!

Reply:
Yes, or just a text version of that screenshot, in a frame. something like

> `@PR Reviews` this pull request to do XYZ 
> PR: [link to GH PR](https://github.com/bcgov/cas-home/pull/8) 
> Card: ...

etc